source/frontend/src/features/admin/edit-user/EditUserContainer.tsx

```diff
@@ -50,17 +51,17 @@ const EditUserContainer: React.FunctionComponent<IEditUserContainerProps> = ({ u
     regions: [],
     note: '',
     position: '',
-    userTypeCode: { id: UserTypes.Contractor },
+    userTypeCode: toTypeCodeNullable(UserTypes.Contractor) ?? undefined,
```

Comment:
seems a bit verbose. is there a non-null version of toTypeCode that we could use here instead since we are passing a constant?

Reply:
good catch